src/networking.rs

```diff
@@ -37,12 +37,16 @@ impl TcnApi for TcnApiImpl {
   }
 
   fn post_report(&self, report: String) -> Result<(), NetworkingError> {
+    println!("RUST posting report: {}", report);
```

Comment:
@i-schuetz Should we look into logging solution on Rust side?

Reply:
Yeah, I actually started writing an issue about it a couple of days ago, but didn't want to distract from more urgent things. It would be a wrapper like we've on Android. At some point we'd should able to pass a callback to Rust in order to send the logs to the apps (e.g. to display on the screen).

Reply:
If you want to do it (for now the callback isn't necessary), go ahead! It shouldn't be actually that much work.